### PR TITLE
Fix several typos

### PR DESCRIPTION
**Pull request by nnsW3**
Hello there,
I managed to fix several minor typos found in your documentation, hope it helps.
Best regards, Elias.

### CODE REVIEW

File: docs/src/concepts/channels.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The `latest` channel is pointing to our latest beta network. This toolchain shou
 
 > **Note**
 >
-> The `latest` channel used to point latest compatible versions of `forc` and `fuel-core`, after version v0.20.0 latest is changed to point to the latest network. This is a breaking change and should be taken into account for existing workflows.
+> The `latest` channel is used to point latest compatible versions of `forc` and `fuel-core`, after version v0.20.0 latest is changed to point to the latest network. This is a breaking change and should be taken into account for existing workflows.
```
Comment (sdankel) [inline docs/src/concepts/channels.md:48]: It was correct before.

File: docs/src/developer_guide/contributing_to_fuelup.md [hunk 2]
```diff
@@ -44,20 +44,20 @@ There are many ways in which you may contribute to `fuelup`, some of which invol
 
 Check out our [Help Wanted](https://github.com/FuelLabs/fuelup/labels/help%20wanted), [Fuelup Book](https://github.com/FuelLabs/fuelup/labels/book) or [Good First Issue](https://github.com/FuelLabs/fuelup/labels/good%20first%20issue) issues to find a suitable task.
 
-If you are planning something big, for example, related to multiple components or changes current behaviors, make sure to open an issue to discuss with us before starting on the implementation.
+If you are planning something big, for example, related to multiple components or changes in current behaviors, make sure to open an issue to discuss with us before starting on the implementation.
```
Comment (sdankel) [inline docs/src/developer_guide/contributing_to_fuelup.md:47]:
```suggestion
If you are planning something big, for example, related to multiple components or changes to current behaviors, make sure to open an issue to discuss with us before starting on the implementation.
```

File: docs/src/developer_guide/contributing_to_fuelup.md [hunk 3]
```diff
@@ -44,20 +44,20 @@ There are many ways in which you may contribute to `fuelup`, some of which invol
...
 
 ## Contribution flow
 
 This is a rough outline of what a contributor's workflow looks like:
 
 - Make sure what you want to contribute is already tracked as an issue.
-  - We may discuss the problem and solution in the issue.
+  - We may discuss the problem and solution to the issue.
```
Comment (sdankel) [inline docs/src/developer_guide/contributing_to_fuelup.md:54]: Correct before

**Review by sdankel — CHANGES_REQUESTED** [review]

**Comment by nnsW3** [comment]
solved, hope it helps

**Comment by sdankel** [comment]
You'll need to fix the conflicts with master.

**Comment by nnsW3** [comment]
done

File: docs/src/developer_guide/building_a_channel.md [hunk 4]
```diff
@@ -28,7 +28,8 @@ An example of this usage is in [`publish-nightly-channel.yml`].
 
 ### Manual
 
-There may be times where we need a channel for a one-off event e.g. testnets. During these events, we do not
+There may be times when we need a channel for a one-off event eg. testnets. During these events, we do not
```
Comment (sdankel) [inline docs/src/developer_guide/building_a_channel.md:31]:
```suggestion
There may be times when we need a channel for a one-off event e.g. testnets. During these events, we do not
```

**Review by sdankel — APPROVED** [review]

**Pull request merged**